### PR TITLE
Successfully updated the copyright year in the footer and updated the mobile design of Login/Register page

### LINKED ISSUE #166: Update Copyright year
## Describe the bug
eos6

We just need to change the copyright year from 2020 to 2022 which is written on the sign-up card.

### PR DESCRIPTION
**Pull request by dikshashukla455**
The Issue number: #166 and #184
Fixed #166 and #184

I have changed the copyright year from 2020 to 2022 in the footer section. You can see the change on "sign-in" page.
I have also updated the mobile design of the "sign-in" page.

Preview Deployed link: http://localhost:3000/login

- [x] My code follows the code style of this project.
- [x]  All new and existing tests passed.
- [x]  The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by nishu-murmu** [comment]
There is better solution to this! Instead of updating `common.json` file for each year. Why not just use `getFullYear()` function.

**Comment by dikshashukla455** [comment]
> There is better solution to this! Instead of updating `common.json` file for each year. Why not just use `getFullYear()` function.

Yeah! getFullYear() function is better option  but I didn't get it into my mind at that point of time when I was working on this issue. That's why, I updated the common.json file. But I used this function in future. Thank you

**Comment by nishu-murmu** [comment]
> > There is better solution to this! Instead of updating `common.json` file for each year. Why not just use `getFullYear()` function.
> 
> Yeah! getFullYear() function is better option but I didn't get it into my mind at that point of time when I was working on this issue. That's why, I updated the common.json file. But I used this function in future. Thank you

You can update your PR?!

**Comment by cyntss** [comment]
I agree with @nishu-murmu . If there is a better solution and the PR is still open, better to solve it and not waste time reviewing 2 PRs :)

**Comment by dikshashukla455** [comment]
> > > There is better solution to this! Instead of updating `common.json` file for each year. Why not just use `getFullYear()` function.
> > 
> > 
> > Yeah! getFullYear() function is better option but I didn't get it into my mind at that point of time when I was working on this issue. That's why, I updated the common.json file. But I used this function in future. Thank you
> 
> You can update your PR?!

I will update it today. Thank you

**Comment by dikshashukla455** [comment]
> I agree with @nishu-murmu . If there is a better solution and the PR is still open, better to solve it and not waste time reviewing 2 PRs :)

Yeah, I will update my PR.

**Comment by dikshashukla455** [comment]
> > I agree with @nishu-murmu . If there is a better solution and the PR is still open, better to solve it and not waste time reviewing 2 PRs :)
> 
> Yeah, I will update my PR.

I have updated the Pull Request having with a better solution. Kindly review it. Thank you for your time

File: .gitignore [hunk 1]
```diff
@@ -9,8 +9,8 @@
 /coverage
 
 # production
-/build
-/src/config.json
+
```
Comment (cyntss) [inline .gitignore:12]: please don't change the gitignore

File: src/assets/scss/components/dropdown.scss [hunk 2]
```diff
@@ -3,29 +3,20 @@
   display: inline-block;
   padding: 0px 8px;
   position: relative;
-
-  .eos-icons {
+  
+  .eos-icons1 {
```
Comment (cyntss) [inline src/assets/scss/components/dropdown.scss:7]: don't change the the class name and please don't remove the properties either. The `eos-icons` class is the default for the icons.

Additionally, never use numbers in your classes, this is a very bad practice. Classes need to reflect their utility, not a version control system for classes.

File: src/assets/scss/components/dropdown.scss [hunk 3]
```diff
@@ -102,3 +93,20 @@
     padding: 0px;
   }
 }
+  @media only screen and (min-width: 360px) {
+    .eos-icons1 {
```
Comment (cyntss) [inline src/assets/scss/components/dropdown.scss:97]: same issues here. The original class must not be changed.

File: src/assets/scss/pages/authentication.scss [hunk 4]
```diff
@@ -179,6 +179,10 @@
     border-top: 1px solid $border-gray;
     padding: 16px 0;
   }
+ .image-logo{
+ width: 120px !important;
```
Comment (cyntss) [inline src/assets/scss/pages/authentication.scss:183]: please check your indentation

File: src/assets/scss/pages/authentication.scss [hunk 5]
```diff
@@ -196,7 +200,14 @@
     z-index: 1;
   }
 }
-
+@media only screen and (max-width: 360px) {
+  .image-logo{
+    width: 95px !important;
```
Comment (cyntss) [inline src/assets/scss/pages/authentication.scss:205]: indentation is incorrect

File: src/components/LanguageDropdown.js [hunk 6]
```diff
@@ -27,15 +27,15 @@ const LanguageDropdown = (props) => {
     <div className='dropdown-container' ref={container}>
       <Button
         type='button'
-        className='btn btn-secondary flex flex-row'
+        className='btn btn-secondary flex'
         onClick={handleButtonClick}
       >
         {dropdownState ? (
-          <EOS_KEYBOARD_ARROW_UP className='eos-icons' />
+          <EOS_KEYBOARD_ARROW_UP className='eos-icons1' />
```
Comment (cyntss) [inline src/components/LanguageDropdown.js:34]: please revert to the original class name

**Comment by cyntss** [comment]
To be honest, I would prefer a cleaner solution and a separation of the 2 issues. Can you please close this PR and open a new one where you only introduce the dateTime() method for the year automation?

The work on the UI should be treated in a different PR.

**Comment by dikshashukla455** [comment]
Sorry, for these mistakes. 

> To be honest, I would prefer a cleaner solution and a separation of the 2 issues. Can you please close this PR and open a new one where you only introduce the dateTime() method for the year automation?
> 
> The work on the UI should be treated in a different PR.

Sorry for all these mistakes, ,I will update my two separate Pull request and closing this one .
Thank you.

**Pull request closed without merging**